test/test_apps/triangle/app.cpp

```diff
@@ -0,0 +1,498 @@
+/*
+** Copyright (c) 2018-2024 LunarG, Inc.
+**
+** Permission is hereby granted, free of charge, to any person obtaining a
+** copy of this software and associated documentation files (the "Software"),
+** to deal in the Software without restriction, including without limitation
+** the rights to use, copy, modify, merge, publish, distribute, sublicense,
+** and/or sell copies of the Software, and to permit persons to whom the
+** Software is furnished to do so, subject to the following conditions:
+**
+** The above copyright notice and this permission notice shall be included in
+** all copies or substantial portions of the Software.
+**
+** THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+** IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+** FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+** AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+** LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+** FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+** DEALINGS IN THE SOFTWARE.
+*/
+
+#include <iostream>
+
+#include <vulkan/vulkan_core.h>
+
+#include <test_app_base.h>
+
+#include <SDL3/SDL_main.h>
+
+GFXRECON_BEGIN_NAMESPACE(gfxrecon)
+
+GFXRECON_BEGIN_NAMESPACE(test_app)
+
+GFXRECON_BEGIN_NAMESPACE(triangle)
+
+const int MAX_FRAMES_IN_FLIGHT = 2;
```

Comment:
nit: I would use `size_t` here as it is used for sizing the `command_pools` array and `current_frame` is `size_t` as well.